Etapa1/src/com/company/Main.java

```diff
@@ -0,0 +1,11 @@
+package com.company;
```

Comment:
main e foarte bun, great job, doar ca folosesc clasa asta ca sa-ti las feedback in legatura cu ce trebuia sa adaugi la .gitignore pentru a nu adauga si codul compilat de java sau configurarile de intellij. fisierul Etapa1.imp nu trebuia inclus, si nici tot pachetul out. Also, trebuia sa ai proiect/repo nou, separat, pentru acest proiect.

Reply:
![image](https://user-images.githubusercontent.com/67580205/161250647-ab44641a-4884-417a-9711-90deda1a1ab6.png)
mentionez aici ca daca bagi input gresit, crapa. nu presupune niciodata ca userul va folosi aplicatia cum vrei tu, ei vor gasi mereu tampenii pe care sa le faca :)))

Reply:
Stiu, doar ca aveam extrem de multe cazuri de input gresit. Am incercat sa tratez cat mai multe, dar nu am avut timp pentru toate.